kotlinsample/src/main/java/com/airbnb/epoxy/kotlinsample/models/ItemViewBindingEpoxyHolder.kt

```diff
@@ -0,0 +1,20 @@
+package com.airbnb.epoxy.kotlinsample.models
+
+import com.airbnb.epoxy.EpoxyAttribute
+import com.airbnb.epoxy.EpoxyModelClass
+import com.airbnb.epoxy.kotlinsample.R
+import com.airbnb.epoxy.kotlinsample.databinding.ViewBindingHolderItemBinding
+import com.airbnb.epoxy.kotlinsample.helpers.ViewBindingEpoxyModel
+
+@EpoxyModelClass(layout = R.layout.view_binding_holder_item)
+abstract class ItemViewBindingEpoxyHolder :
+        ViewBindingEpoxyModel<ViewBindingHolderItemBinding>(ViewBindingHolderItemBinding::bind) {
```

Comment:
I think you can avoid passing `ViewBindingHolderItemBinding::bind` by using reflection inside `ViewBindingEpoxyModel` to get the static `bind` function. This would be a nicer user experience. That reflected method lookup can be cached so it is only done once for each class of viewholder (or done in a map to do once per binding type)

Reply:
Implemented the reflection approach, a bit more verbose/cryptic and with 2 cache levels (EpoxyModel via lazy to retrieve the generics class info + Static for caching the binding method given a specific class).

Looks clearly better for the user experience.